docs/how_to/install.rst

```diff
@@ -88,11 +88,13 @@ INSTALLED_APPS
 
 You will need to add the following to its list of ``INSTALLED_APPS``::
 
+    'myproject'
```

Comment:
isn't this already the case in the standard `settings.py` generate by django-admin?

Reply:
It isn't in Django 3.1 or 2.2. Not sure about other versions.